README.md

```diff
@@ -41,7 +41,7 @@ No matter what category of app you have, you'll need your Intercom app id and th
 
 	- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
 		// Initialize Intercom
-    	[Intercom setApiKey:@"<#ios_sdk-...#>" forAppId:@"<#your-app-id#>"];
+    	[Intercom setApiKey:@"ios_sdk-<#your-api-key#>" forAppId:@"<#your-app-id#>"];
```

Comment:
Thanks @bnd5k! +1 @jtreanor. 

This prefix is actually part of the API Key so if you were to tab to this placeholder having copied it out of the integration settings you might accidentally double up on the prefix. I worry that by suggesting it's outside the scope of pasting here, people may assume that any other API key (from another integration perhaps) might work. `<#your_ios_api_key#>` might indeed be better.